### DISCUSSION TITLE
Is that possible to export postgres query data to csv in parallel?

### DISCUSSION THREAD

**Discussion by dawsondong** [duckdb/duckdb#7882] [General]
If we use duckdb postgres_scanner to connect to postgres, we want to take advantage of its parallel scanner to bring results into a csv file. Is it possible and how to do it?

**Comment by l1t1** [comment]
https://duckdb.org/docs/sql/statements/copy#csv-export

per_thread_output	Generate one file per thread, rather than one file in total. This allows for faster parallel writing.
